nats.go

```diff
@@ -266,8 +269,15 @@ type Options struct {
 
 	// DisconnectedCB sets the disconnected handler that is called
 	// whenever the connection is disconnected.
+	// Will not be called if DisconnectedErrCB was initiated.
```

Comment:
should it be: `Will not be called if DisconnectedErrCB is set`

Reply:
thanks, fixed docs